src/ACEtk/details/verify/src/sorted.hpp

```diff
@@ -0,0 +1,15 @@
+template< typename Range,
+          utility::Require< false, IsSorted, Range > = true,
+          utility::Require< true, utility::is_range, Range > = true 
+        >
+auto sorted( Range&& r ){
+  return Sorted< std::decay_t< Range > >{ std::forward< Range >( r ) };
+}
+
+template< typename Range,
+          utility::Require< false, IsSorted, Range > = true 
```

Comment:
the is only safe if the range is sorted. Here you're enforcing that it's not sorted.

Reply:
Good catch. Another copy/paste error.